### ISSUE TITLE
Problem with tracing a program

### ISSUE THREAD

**Issue by salmivas** [crystalidea/uninstall-tool#28]
Hello.

I've been using Uninstall Tool for a few months end everything was OK. But about a week ago I faced a problem. When I'm trying to install any program by using "Install and Trace" installation crashes with error message. 

Log is below:
10:55:54        Starting Install Tracker service...
10:55:55        Service version: 0x105
10:55:55        Starting 'Listary.exe'...
10:55:55        Error: Cannot start the setup process (error 193)
10:56:00        Error: Failed to stop the tracking service

**Comment by kleuter** [comment]
Hi,

Looks like this problem is known, and it's not UT fault:
https://forums.crystalidea.com/viewtopic.php?t=440&start=20#p4187

**Comment by salmivas** [comment]
I solved the problem just switching off the service below:
![image](https://user-images.githubusercontent.com/35002517/49684153-44006280-fae9-11e8-947d-56fefea4a2e7.png)
After restart and till now everything is OK.

**Comment by kleuter** [comment]
To anyone who uncounted the issue, does turning this option off helps?

![sshot-1](https://user-images.githubusercontent.com/2600624/68202257-9d0b9b80-ffc3-11e9-88d9-4cad2af5053a.png)

**Comment by The-alchimist** [comment]
Yes it help, you have work man because i bought your software , and it's a great software to by the way,...

Verry usefull for analysing malware ^^, better that other uninstaller who has crapware and bundelware and other extra feature that is not the priority or even required.

Ps: i recommend you to stay focus on uninstalling and installing nothing else, no extra feature but just bypass all windows control program for my opinion it's enough.

Have a nice day.
